src/containers/Search/hooks/shared.ts

```diff
@@ -0,0 +1,26 @@
+export function merge(fromArr: SearchItem[], toArr: SearchItem[]) {
```

Comment:
need to care about more/less clean architecture. I suggest to move such functions to separate module, if it exported - reused somwhere else

Reply:
it may be temp code, don't like how oracle requests are doing, we will continue update it next PRs